### ISSUE TITLE
Panscrolling doesn't work in QT apps

### ISSUE THREAD

**Issue by Greenscreener** [linuxwacom/xf86-input-wacom#247]
This is another issue I found while working on #222. Some debug logs can be found [here](https://github.com/linuxwacom/xf86-input-wacom/pull/222#issuecomment-1030687835).
I'll look further into it and bisect it tomorrow.

**Comment by andrejp88** [comment]
I've been using my tablet more recently, and have noticed this problem with Qt apps. I think I've tested every Qt app I have installed.

The Dolphin Emulator (`dolphin-emu` in Arch), which uses Qt6, does not have this problem. However, qBittorrent and Stellarium, my only other Qt6 apps, *do* have the problem.

All other Qt-based apps on my computer depend on Qt5, not 6, and all of them have the scrolling issue, including Dolphin (the file explorer), system settings, and Plasma Desktop stuff such as system tray popups.

For some reason, the Dolphin Emulator does something different here.

Not sure if this helps at all but I thought it would be a bit more useful than just a thumbs up :stuck_out_tongue:
